phpunit.xml.dist

```diff
@@ -14,4 +14,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <logging>
+        <log type="coverage-clover" target="coverage.xml"/>
```

Comment:
This will increase the time to run unit tests. A better solution would be to only run coverage on a specific test. For reference: https://github.com/php-http/HttplugBundle/blob/master/.travis.yml

But I think this is fine for now.